Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-05-16
Initial Package Version: 133mr
Origin: Jim Gifford
Description: Fixes GCC 3.3 Errors

diff -uNr pccts.orig/sorcerer/lib/sintstack.c pccts/sorcerer/lib/sintstack.c
--- pccts.orig/sorcerer/lib/sintstack.c	1999-06-30 07:08:06.000000000 -0700
+++ pccts/sorcerer/lib/sintstack.c	2003-05-16 19:50:11.000000000 -0700
@@ -29,11 +29,7 @@
 #include <stdio.h>
 #include <setjmp.h>
 
-#ifdef PCCTS_USE_STDARG
 #include <stdarg.h>
-#else
-#include <varargs.h>
-#endif
 
 #include "CASTBase.h"
 #include "sintstack.h"
diff -uNr pccts.orig/sorcerer/lib/sorlist.c pccts/sorcerer/lib/sorlist.c
--- pccts.orig/sorcerer/lib/sorlist.c	1999-06-29 19:15:56.000000000 -0700
+++ pccts/sorcerer/lib/sorlist.c	2003-05-16 19:50:22.000000000 -0700
@@ -29,11 +29,7 @@
 #include <stdio.h>
 #include <setjmp.h>
 
-#ifdef PCCTS_USE_STDARG
 #include <stdarg.h>
-#else
-#include <varargs.h>
-#endif
 
 #include "CASTBase.h"
 #include "sorlist.h"
